Contributing.md.txt

```diff
@@ -0,0 +1,113 @@
+# Contributing to CONTRIBUTING.md
```

Comment:
It looks like there are several places that the contributing file used as a template need updating - primarily updating the places where the project name appears to be CONTRIBUTING.md to Surfactant.